internal/cache/timeoutCache.go

```diff
@@ -115,17 +117,26 @@ func (c *TimeoutCache) removeInternal(key interface{}, runCallback bool) (*cache
 		// of deleted and return.
 		entry.deleted = true
 	}
-	if runCallback {
-		entry.callback()
-	}
 	return entry, true
 }
 
 // Clear removes all entries, and runs the callbacks if runCallback is true.
 func (c *TimeoutCache) Clear(runCallback bool) {
+	var entries []*cacheEntry
```

Comment:
So `entries` will always be an empty list? Then the for loop on L139 is dead code.